src/Workspaces/Core/Desktop/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+static Microsoft.CodeAnalysis.Host.Mef.DesktopMefHostServices.DefaultAssemblies.get -> System.Collections.Immutable.ImmutableArray<System.Reflection.Assembly>
```

Comment:
Yes.  It is already public in MefHostServices type.  It is typically used when you want to create a different host service instantiation by just added a few more assemblies to the mix.